### DISCUSSION TITLE
能否提供上传文件的API

### DISCUSSION THREAD

**Discussion by 276523923** [AlistGo/alist#1025] [Ideas]
用户后台生成一个上传token，然后调用API上传文件。
自己写的一个app，想调用接口上传文件。

**Comment by xhofe** [comment]
### API
- token: https://alist-doc.nn.ci/docs/driver/alist#token
- pathname: `/api/public/upload`
- `Content-Type`: `multipart/form-data`
### Form
- files: 文件
- path: 上传的路径

**Reply by 276523923** [reply]
原来是有的啊，我看API文档没看到。
谢谢了我去试试。

**Reply by johnseen** [reply]
@276523923 测试过了吗? 我用curl一直报错wrong password 403

**Reply by YuChaoM** [reply]
这个接口是可以的，我测试过了

**Comment by Liner03** [comment]
兄弟搞定没，我写的总是403

**Reply by YuChaoM** [reply]
可以呀，你参数没写对，在headers中填写Content-Type=multipart/form-data还有token

**Reply by YuChaoM** [reply]
![image](https://user-images.githubusercontent.com/101857574/179272954-d29d8f7c-b44d-48c0-981e-f5c5dd63719b.png)

**Reply by Liner03** [reply]
能留个V吗，方便联系下
